### ISSUE TITLE
Set telegraf.exe Property Details to include version number

### ISSUE THREAD

**Issue by aslgithub** [influxdata/telegraf#9385]
## Feature Request

The telegraf.exe properties for the version are empty so beyond the file size and date modified its impossible to tell which version of telegraf.exe without running it.

### Proposal:

Include at least the version number in the Properties.
This stackoverflow question shows the issue 
https://stackoverflow.com/questions/35126344/how-to-set-the-file-version-field-in-pe-header-of-exe-files
Which proposes two different solutions to including the required manifest on the windows build.
https://github.com/josephspurrier/goversioninfo
https://github.com/tc-hib/go-winres

### Current behavior:

On Windows the telegraf.exe file Property Details are not populated.

### Desired behavior:

The file Property Details are included so a version number can be seen.

### Use case: 
It's impossible beyond the file size and date modified to tell which release of Telegraf.exe is without running it.

**Comment by powersj** [comment]
@sspaink didn't you do this already?

**Comment by sspaink** [comment]
yup! https://github.com/influxdata/telegraf/pull/10487 this will be released with v1.22, I don't it is in v1.21 yet.
